### ISSUE TITLE
Errors when browsing to folders with empty spaces in its name, breadcrumb

### ISSUE THREAD

**Issue by luismesagravedeperalta** [coderaiser/cloudcmd#171]
Hi, when browsing back through the breadcrumb to folders with empty spaces in its name, get the following error:
ENOENT: no such file or directory, scandir 'root/cloudcmddata/20161206 Les deux alpes/'

**Comment by luismesagravedeperalta** [comment]
Al latest, from yesterday (Running on docker on a RPi). All very simple setup.

This is what I'm doing:
Dockerfile:
# Use the official node as a parent image
FROM node
# Install cloud commander package 
RUN npm i cloudcmd -g
# Make cloudcmd port 8000 available to the world outside this container
EXPOSE 8000
# Run app
CMD ["cloudcmd"]



docker-compose:
version: '2'

services:
  app:
    image: cloudcmdimg
    ports:
      - 8080:8000
    volumes:
      - /home/pi/sambashare:/root/cloudcmddata
    environment:
      - CLOUDCMD_ROOT=root/cloudcmddata
    restart: always

**Comment by coderaiser** [comment]
Did you clear cache of your browser?

**Comment by luismesagravedeperalta** [comment]
Reproduced in demo site: https://drive.google.com/open?id=1a9tgIPqxi4QVCC2Ppwew6JRETOKpUqgg

**Comment by coderaiser** [comment]
Why do you create your own `Dockerfile`? You can use one of this:
- `coderaiser/cloudcmd:latest`
- `coderaiser/cloudcmd:latest-alpine`

from [dockerhub](https://hub.docker.com/r/coderaiser/cloudcmd/tags/)

Or if you create it anyway you can try suggested in #73 way, but using [arv32v7/node](https://hub.docker.com/r/arm32v7/node/) instedof [hypriot/rpi-node](https://hub.docker.com/r/hypriot/rpi-node/) because it is deprecated:
```
FROM arm32v7/node
RUN npm i cloudcmd -g
EXPOSE 8000
CMD ["cloudcmd"]
```

**Comment by coderaiser** [comment]
> Reproduced in demo site: https://drive.google.com/open?id=1a9tgIPqxi4QVCC2Ppwew6JRETOKpUqgg

OK, I got it, you are talking about the link in the path. I'll check it, thank you :).

**Comment by coderaiser** [comment]
Thank you :), fixed with 753db12, landed in [v9.3.1](https://github.com/coderaiser/cloudcmd/releases/tag/v9.3.1).
